### ISSUE TITLE
ADAM Optimizer scheduler

### ISSUE THREAD

**Issue by Andrewblumensen** [ES0R/mlops_project#16]
We need to introduce an ADAM optimizer for the training loop

**Comment by Andrewblumensen** [comment]
Solved
